ipfs-lite/ipfs_lite.proto

```diff
@@ -0,0 +1,218 @@
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+option java_multiple_files = true;
+option java_package = "io.textile.grpc_ipfs_lite";
+option java_outer_classname = "IpfsLiteProto";
+
+// Model messages
+
+message AddParams {
+    string layout = 1;
+    string chunker = 2;
+    bool rawLeaves = 3;
+    bool hidden = 4;
+    bool shared = 5;
+    bool noCopy = 6;
+    string hashFun = 7;
+}
+
+message Block {
+    bytes rawData = 1;
+    string cid = 2;
+}
+
+message Link {
+    string name = 1;
+    int64 size = 2;
+    string cid  = 3;
+}
+
+message NodeStat {
+    string hash = 1;
+    int32 numLinks = 2;
+    int32 blockSize = 3;
+    int32 linksSize = 4;
+    int32 dataSize = 5;
+    int32 cumulativeSize = 6;
+}
+
+message Node {
+    Block block = 1;
+    repeated Link links = 2;
+    NodeStat state = 3;
+    int64 size = 4;
+}
+
+// Request and Response messages
```

Comment:
These are request/response wrapper messages to help with compatibility when making api changes in the future. Good blog post here about best practices https://medium.com/@akhaku/protobuf-definition-best-practices-87f281576f31